i18n/src/main/resources/openecard_i18n/pinplugin/Messages_de.properties

```diff
@@ -10,6 +10,7 @@ action.changepin.description = Mit dieser Aktion können Sie Ihre PIN ändern.
 action.changepin.userconsent.title = PIN ändern - %s
 
 action.changepin.userconsent.pinstep.title = PIN-Eingabe
+action.changepin.userconsent.pinstep.context_specific = Signatur PIN-Eingabe
```

Comment:
Actually its the Signature-PIN which is entered. This is also wrong in the above translation.
And don't forget the french translation ;-)=